source/Preferences.cpp

```diff
@@ -153,6 +155,7 @@ void Preferences::Load()
 	// These settings should be on by default. There is no need to specify
 	// values for settings that are off by default.
 	settings["Landing zoom"] = true;
+	settings["Show parenthesis on Job Board"] = false;
```

Comment:
no need to set a default value for preferences defaulting to "off"
```suggestion
```

Reply:
Hm, shouldn't this default to `true` to match the current default?

Reply:
The lack of parenthesis is the more desirable behavior, see #9203.